src/models/commentModel.js

```diff
@@ -1,14 +1,29 @@
 import mongoose from 'mongoose';
+const { Schema } = mongoose;
 
 const CommentSchema = new mongoose.Schema({
-  id: {
+  content: {
     type: String,
-    required: true,
-    unique: true
+    required: true
   },
-  commenterId: String,
-  content: String,
-  date: Date
+  commenterName: {
```

Comment:
are we able to get the name from the authorID. you know, single source of truth?